### ISSUE TITLE
No rule to make target `install`

### ISSUE THREAD

**Issue by johnjelinek** [alisaifee/asdf-thrift#1]
```
$ asdf install thrift latest

Downloading source.....
Configuring build....
Compiling with flags -j10....
make: *** No rule to make target `install'.  Stop.
asdf-thrift: An error ocurred while installing thrift 0.16.0.
```

**Comment by johnjelinek** [comment]
Note: I'm on an M1 mac

**Comment by alisaifee** [comment]
Sorry for the huge delay - is this still a problem? I unfortunately don't have access to an M1 to reproduce

**Comment by haocheng** [comment]
@alisaifee Yes, I saw the same errors on M1 MacBook Pro, too...

```
> asdf install thrift latest
Downloading source.....
Configuring build....
Compiling with flags -j10....
make: *** No rule to make target `install'.  Stop.
asdf-thrift: An error ocurred while installing thrift 0.16.0.
```

**Comment by durandj** [comment]
I'm getting the same thing on an Ubuntu machine.

```
$ asdf install thrift latest
Downloading source.....
Configuring build....
Compiling with flags -j20....
make: *** No rule to make target 'install'.  Stop.
asdf-thrift: An error ocurred while installing thrift 0.17.0.
```

**Comment by durandj** [comment]
Did a bit of debugging on this.

I first removed the `> /dev/null 2>&1` on the `./configure` call so that I can check if there's something failing during that that I was missing (maybe a dependency or something) and got `./configure: No such file or directory`.

I then added logging back in for the `./bootstrap.sh` call and that showed that I was missing `libtoolize`. So after installing that I still got failures when building for things like `flex` missing.

I eventually got it working but it might be worthwhile from a debugging perspective to either not redirect stderr to `/dev/null` or to make it so that a user can override that behavior in configuration instead of code to make it easier to fix these kinds of issues.
